widget/check_group.go

```diff
@@ -44,6 +45,24 @@ func (r *CheckGroup) Append(option string) {
 	r.Refresh()
 }
 
+// Remove removes the first specified option found from a CheckGroup widget.
+//
+// Since: 2.3
+func (r *CheckGroup) Remove(s string) bool {
+	found := -1
+	for i, o := range r.Options {
+		if strings.EqualFold(s, o) {
+			found = i
+			continue
```

Comment:
This seems like it should be a `break`. However, please see my other comment about this not being necessary.

Reply:
Oh my... Of course, it is a break...